packages/integrations/node/src/types.ts

```diff
@@ -15,6 +16,7 @@ export interface Options extends UserOptions {
 	port: number;
 	server: string;
 	client: string;
+	trailingSlash: AstroUserConfig['trailingSlash'];
```

Comment:
I don't think we should change the signature of the Node options. This information is already available in the Astro config, so it should be available to the `app` that is generated by Astro.

Reply:
Yeah, agree here